js/ui/grid_core/ui.grid_core.accessibility.js

```diff
@@ -1,6 +1,6 @@
 import * as accessibility from '../../ui/shared/accessibility';
 
-module.exports = {
+export default {
     registerKeyboardAction: function(viewName, instance, $element, selector, action) {
```

Comment:
I think we should export registerKeyboardAction function instead of json with this function